### DISCUSSION TITLE
OSX-PROXMOX – I did not know ...

### DISCUSSION THREAD

**Discussion by lexybarton** [tteck/Proxmox#1691] [General]
[OSX-PROXMOX](https://github.com/luchina-gabriel/OSX-PROXMOX) 
‼ the oneliner will modify your grub and reboot – no questions asked! quite BRUTAL 🔪🩸💀 (god bless @tteck 🙏)

anyway, it does what it promises, so I wanted to share it, maybe someone might find it useful and had no clue it's possible before (I just found you can run OSX even in docker these days... 😵)

![image](https://github.com/tteck/Proxmox/assets/14018895/fb341ae9-d444-45e5-b840-a71d16e26f93)

**Comment by aytec** [comment]
did you try it with an existend proxmox server?
what to do if i decide zu rollback? :)
i ran the command, proxmox rebooted (saw to late that it would change something with the proxmox configuration itself).. my other vms ansd LXC runs fine, but havent stanrted the part after the reboot..... :)

**Comment by Deekerman** [comment]
I have tried it multiple times and never got it to work. Never got past the screen in your picture. Always stopped. This would be an amazing Script but seems less likely.
